t/lib/MT/Test/Fixture.pm

```diff
@@ -480,9 +480,12 @@ sub prepare_template {
             my %arg
                 = ref $item eq 'HASH' ? %$item : ( archive_type => $item );
             my $archive_type = delete $arg{archive_type};
-            my $archiver     = MT->publisher->archiver($archive_type)
-                or croak "unknown archive_type: $archive_type";
-            $arg{type} = _template_type($archive_type);
+            my $archiver;
+            if ($archive_type) {
+                $archiver = MT->publisher->archiver($archive_type)
+                    or croak "unknown archive_type: $archive_type";
+                $arg{type} = _template_type($archive_type);
+            }
```

Comment:
@taitetsu If $archiver is empty, there should be no $mapping and thus that part shouldn't be reached because of ```next unless $mapping```, but it's certainly better to make it clearer. Will fix it. Thanks.

Reply:
@taitetsu Done. Could you please review the last commit?

Reply:
looks good for me.